leo/commands/gotoCommands.py

```diff
@@ -25,36 +25,38 @@ def __init__(self, c: Cmdr) -> None:
 
     #@+others
     #@+node:ekr.20100216141722.5622: *3* goto.find_file_line
-    def find_file_line(self, n: int, p: Position = None) -> Tuple[Position, int, bool]:
+    def find_file_line(self, n: int, p: Position = None) -> Tuple[Position, int]:
         """
         Place the cursor on the n'th line (one-based) of an external file.
-        Return (p, offset, found) for unit testing.
+
```

Comment:
This doc string should point out the return values for a failure to find the line is a (None, -1) tuple.